### DISCUSSION TITLE
Distinguish better heat pumps from boilers

### DISCUSSION THREAD

**Discussion by proddy** [emsesp/EMS-ESP32#1841] [Ideas]
Modern heat pumps are categorized as Boilers in ESM-ESP (DeviceType::BOILER) because the DeviceID is 8 and it uses many of the EMS+ telegrams from a standard boiler.  See how the "Bosch Compress 7000i AW Heat Pump" is shown in the demo https://ems-esp.derbyshire.nl/devices

I was thinking, for starters, if a device has the flag `EMS_DEVICE_FLAG_HEATPUMP` set could we set the device type to Heatpump in the WebUI, so at least it doesn't show as a Boiler in the Web (name + icon). However, there are other places where this will be impacted, and doing it properly breaks things.

@MichaelDvP what do you think?

**Comment by MichaelDvP** [comment]
I think it's confusing to show in dashboard "Heatpump", but the entities are published in boiler_data, and commands in mqtt/api/telnet have to be `boiler/<entitiy>`. Maybe change test in dashboard to `HP/Boiler` or `Boiler/HP`

**Reply by proddy** [reply]
that could work. If the flag EMS_DEVICE_FLAG_HEATPUMP is set then append the text "HP", only in the WebUI

It's not important or urgent right now.
